src/Tribe/Views/V2/Hooks.php

```diff
@@ -95,6 +96,20 @@ protected function add_filters() {
 		}
 	}
 
+	/**
+	 * Includes includes edge cases for filtering when we need to manually overwrite theme's read
```

Comment:
* :no_entry_sign: Doc comment for parameter "$file" missing
* :no_entry_sign: Doc comment for parameter "$name" missing
* :no_entry_sign: Doc comment for parameter "$template" missing

<sub>hash: 6f0ec2f7c84daf79f2060c1cba647095</sub>